index.js

```diff
@@ -217,6 +217,73 @@ var MapToNamespace = function(entity, collection) {
   db[collection].push(entity);
 };
 
+var RemoveFromNamespace = function(db, table, collection) {
```

Comment:
Some dead code in here, and it looks like you're never calling this function with the `collection` parameter set -- were you going for dynamically dropping and removing views?